### PR TITLE
Simplify CONTRIBUTING.md for filling an issue

### PR DESCRIPTION
**Pull request by karellm**
Per the discussion in #1411, an attempt to simplify the CONTRIBUTING Writing section for people only filling an issue. I haven't touched the part for contributing to the actual code since I think I'd need to know how you work better before suggesting changes. That said, I think it is also slightly too long and could use some trimming :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,44 +32,16 @@ issue](https://github.com/rails-api/active_model_serializers/issues/new):
...
-Include as much sample code as you can to help us reproduce the issue. (Inline, repo link, or gist, are fine. A failing test would help the most.)
-
-This is extremely important for narrowing down the cause of your problem.
-
 Thanks!
 
-Sometimes an issue will be closed by a maintainer for various reasons.  In some cases, this is
-an invitation to make a better case for your issue or be able to reproduce a bug, and
-its being close is just an opportunity to help out some more, and then re-open.
-
```
Comment (bf4) [inline CONTRIBUTING.md]: I think this is important.  Maybe there's another place for this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,44 +32,16 @@ issue](https://github.com/rails-api/active_model_serializers/issues/new):
...
+- Ruby version: `ruby -e "puts RUBY_DESCRIPTION"`.
+- Steps to reproduce the issue (i.e. "Show me how to show myself." ). What did you expect to happen? What happened? What did you try?
 
 Simon Tatham has written an excellent on article on
 [How to Report Bugs Effectively](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
 which is [well worth reading](http://yourbugreportneedsmore.info/), although it is not specific to ActiveModelSerializers.
 
-Include as much sample code as you can to help us reproduce the issue. (Inline, repo link, or gist, are fine. A failing test would help the most.)
-
-This is extremely important for narrowing down the cause of your problem.
```
Comment (bf4) [inline CONTRIBUTING.md]: I think this is important.  Maybe there's another place for this?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -32,44 +32,16 @@ issue](https://github.com/rails-api/active_model_serializers/issues/new):
...
-    - The best help here is a failing test. Even better if it's a PR.
-    - Then the steps to reproduce and/or a gist or repository that demonstrates the defect.
-    - Then examples of the code you were using.
-    - Any error messages (including stacktrace, i.e. "Show me the error.")
-  - Things you've tried.
-  - A pull request for your fix would be great.  Code should have tests.
-  - Link to source code, if available.
-
-Please make sure only to include one issue per report.
-If you encounter multiple, unrelated issues, please report them as such.
```
Comment (bf4) [inline CONTRIBUTING.md]: I think this is important.  Maybe there's another place for this?  CONTRIBUTING for overachievers?

**Comment by karellm** [comment]
@bf4  That's why I wasn't sure to be the right person to do this :) 

When editing the document, I basically asked myself "Do I care for that?". If I didn't understand or it felt too specific, I removed it. As I mentioned your content is good. It depends on your goal. 

I targeted first timers. Arguably I would cut it even more because I think it is the beauty of OS not to have to care too deeply about the projects you use. By no mean should users/contributors feel entitled to have their problem resolved but there is a fine line. I do not maintain big projects like this but I accept to have to repeat the "Please provide x, y or z" with whatever I think will be relevant to the issue.

As to your question, you could maybe have a two level document. `CONTRIBUTING.md` would be for first timers and have a link for people that wish to actually write more code for the project.

In all cases, feel free to discard the PR altogether if it is not aligned with what you want.

**Comment by bf4** [comment]
@karellm this is definitely a good issue and worth discussion. Thanks!

**Comment by bf4** [comment]
Oboi, with issue templates I can basically merge this and have my cake too. What do you think of https://github.com/rails-api/active_model_serializers/pull/1530 ?  Comments very much appreciated.

**Comment by karellm** [comment]
That's a nice addition from Github! It moves the pain closer to where the contribution is actually happening. The changes in #1530 look good!

**Comment by remear** [comment]
Thanks @karellm. We've built additional content off this and really appreciate your work.

**Comment by bf4** [comment]
:100:

**Pull request merged**